supervisor/shared/filesystem.c

```diff
@@ -137,11 +141,20 @@ bool filesystem_init(bool create_allowed, bool force_create) {
         // https://specifications.freedesktop.org/trash-spec/trashspec-latest.html
         #endif
 
+        #if CIRCUITPY_SDCARDIO || CIRCUITPY_SDIOIO
+        res = f_mkdir(&vfs_fat->fatfs, "/sd");
+        #if CIRCUITPY_FULL_BUILD
+        MAKE_FILE_WITH_OPTIONAL_CONTENTS(&vfs_fat->fatfs, "/sd/placeholder.txt",
+            "SD cards are mounted at /sd and will shadow this file from Python."
+            " USB will not see the SD card contents here.\n");
+        #endif
+        #endif
+
```

Comment:
I will occasionally mount SD cards to alternate folders so the placeholder message was bothering me a bit. It also took me a couple reads before I understood what the second sentence was trying to say. It's probably just me but what do you think about this as an alternate wording? 

"SD cards mounted at /sd will hide this file from Python."
" SD cards are not visible via USB CIRCUITPY.\n"

Reply:
I like it!